frontend/packages/noobaa-storage-plugin/src/components/resource-providers-card/resource-providers-card-body.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ArrowCircleDownIcon } from '@patternfly/react-icons';
+import { LoadingInline } from '@console/internal/components/utils';
+
+const ResourceProvidersRowStatus: React.FC<ResourceProvidersRowStatusProps> = React.memo(
+  ({ status }) => (
+    <div className="co-resource-providers-card__row-status">
+      <div className="co-resource-providers-card__row-status-item">
+        <div>
+          <ArrowCircleDownIcon />
+        </div>
+        <div className="co-resource-providers-card__row-status-item-text">{status}</div>
+      </div>
+    </div>
+  ),
+);
+
+const ResourceProvidersRow: React.FC<ResourceProvidersRowProps> = React.memo(
+  ({ title, count, unhealthyProviders }) => (
+    <div className="co-resource-providers-card__row">
+      <div className="co-resource-providers-card__row-title">{`${count} ${title}`}</div>
+      {!_.isNil(unhealthyProviders[title]) && unhealthyProviders[title] > 0 ? (
+        <ResourceProvidersRowStatus status={unhealthyProviders[title]} />
+      ) : null}
+    </div>
+  ),
+);
+
+export const ResourceProviderItems: React.FC<ResourceProviderItemsProps> = React.memo(
+  ({ allProviders, unhealthyProviders, isLoading }) => {
+    let render;
+    if (isLoading) render = <LoadingInline />;
```

Comment:
can you add `{` .. `}` for all if/else statements ? it's less error prone to have them for future updates IMO